src/m365/spo/commands/listitem/listitem-attachment-get.ts

```diff
@@ -0,0 +1,132 @@
+import { Logger } from '../../../../cli/Logger.js';
+import GlobalOptions from '../../../../GlobalOptions.js';
+import request, { CliRequestOptions } from '../../../../request.js';
+import { formatting } from '../../../../utils/formatting.js';
+import { urlUtil } from '../../../../utils/urlUtil.js';
+import { validation } from '../../../../utils/validation.js';
+import SpoCommand from '../../../base/SpoCommand.js';
+import commands from '../../commands.js';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  listId?: string;
+  listTitle?: string;
+  listUrl?: string;
+  listItemId: string;
+  fileName: string;
+}
+
+class SpoListItemAttachmentGetCommand extends SpoCommand {
+  public get name(): string {
+    return commands.LISTITEM_ATTACHMENT_GET;
+  }
+
+  public get description(): string {
+    return 'Gets an attachment from a list item';
+  }
+
+  constructor() {
+    super();
+
+    this.#initTelemetry();
+    this.#initOptions();
+    this.#initValidators();
+    this.#initOptionSets();
+  }
+
+  #initTelemetry(): void {
+    this.telemetry.push((args: CommandArgs) => {
+      Object.assign(this.telemetryProperties, {
+        listId: typeof args.options.listId !== 'undefined',
+        listTitle: typeof args.options.listTitle !== 'undefined',
+        listUrl: typeof args.options.listUrl !== 'undefined'
+      });
+    });
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-u, --webUrl <webUrl>'
+      },
+      {
+        option: '--listId [listId]'
+      },
+      {
+        option: '--listTitle [listTitle]'
+      },
+      {
+        option: '--listUrl [listUrl]'
+      },
+      {
+        option: '--listItemId <listItemId>'
+      },
+      {
+        option: '-n, --fileName <fileName>'
+      }
+    );
+  }
+
+  #initValidators(): void {
+    this.validators.push(
+      async (args: CommandArgs) => {
+        const isValidSharePointUrl: boolean | string = validation.isValidSharePointUrl(args.options.webUrl);
+        if (isValidSharePointUrl !== true) {
+          return isValidSharePointUrl;
+        }
+
+        if (args.options.listId && !validation.isValidGuid(args.options.listId)) {
+          return `${args.options.listId} in option listId is not a valid GUID`;
+        }
+
+        if (isNaN(parseInt(args.options.listItemId))) {
```

Comment:
`parseInt` appears to have some issues with parsing but when we type the option as a number then isNaN should suffice.

```suggestion
        if (isNaN(args.options.listItemId)) {
```